### ISSUE TITLE
[css-animations-2] CSS animations on shadow parts

### ISSUE THREAD

**Issue by george-steel** [w3c/csswg-drafts#4716]
`::part()` selectors expose sets of elements in a shadow tree as through a pseudo-element selector on its host. If animation properties are set this way there is some undefined behavior.

Which tree is the @keyframes rule resolved on? Is it the shadow tree of the element or is it the host's tree.

Is the owning element the element in the shadow tree or is it the host? This effects which tree it appears in the `getAnimations()` results for.

**Comment by george-steel** [comment]
@birtles @stephenmcgruer @emilio @graouts

**Comment by emilio** [comment]
This is a dupe of #1516, unless I'm missing something.

**Comment by emilio** [comment]
Which is itself a subset of a more general issue (#1995).

**Comment by birtles** [comment]
Yeah, the question about the owning element might be new, but perhaps that might fall out of the solution to the other issues.
